WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
```

Comment:
Bootstrap doesn't have a variable for this, they just do body { font-size: 14px; }
14px is not that bad, actually. The problem is we're using 1em from https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/assets/stylesheets/navbar-static-top.scss#L2